images.go

```diff
@@ -58,7 +58,7 @@ type ImageUpdateRequest struct {
 // CustomImageCreateRequest represents a request to create a custom image.
 type CustomImageCreateRequest struct {
 	Name         string   `json:"name"`
-	Url          string   `json:"url"`
+	URL          string   `json:"url"`
```

Comment:
I know `golint` reports this as an error, but this is technically a breaking change to this package. Could you revert this change, please?